### PR TITLE
Modify normalisation for namespace support.

### PR DESCRIPTION
**Pull request by davinchia**
## What
Modify `base_normalisation` to better support source defined namespace, specifically allowing normalisation of tables with the same name to different schemas.

## How
* Modify `base_normalisation` internal metadata to also include schema with referencing table names.
* Add `schema` to the file name's dbt generate. Also modify the `ref` macro accordingly. These two pieces of information are used by dbt to generate it's internal dependency graph, so they have to be unique. Lastly, since dbt by default uses the file name as the table's name, use the `alias` [macro](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-aliases) to instruct dbt how actual tables are named.
* Add an acceptance test to confirm this is working as expected.

## Pre-merge Checklist
- [x] *Publish normalisation Docker images* (will do after approval)

## Recommended reading order
1. `stream_processor.py` for the main normalisation changes. `catalog_processor.py` for secondary changes.
1. `test_stream_processor.py` to understand how tests were changed. I did not add additional tests since the current tests do test files are output and named correctly, which is sufficient.
1. `AcceptanceTest` to make sure I didn't make any mistakes.

### CODE REVIEW

**Comment by davinchia** [comment]
Ran manual integration tests to double check this. The postgres source has two tables (`public.films_small` and `staging.films_small`).

Postgres -> Postgres
Screen Shot 2021-04-15 at 8 12 56 PM

Postgres -> Redshift

File: airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java [hunk 1]
```diff
@@ -465,6 +465,27 @@ public void testMultipleSchemasAndTablesSync() throws Exception {
 
   @Test
   @Order(11)
+  public void testMultipleSchemasSameTablesSync() throws Exception {
```
Comment (cgardens) [inline airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java:468]: 🙌

**Review by cgardens — APPROVED** [review]
wooooo! looks good.

can you add a test to `TestDestination` as well to check for the multiple schema case as well please?

**Comment by davinchia** [comment]
@cgardens good call on adding the test - discovered a slight bug in the BigQuery implementation!

**Pull request merged**